config/ftbquests/quests/reward_tables/common.snbt

```diff
@@ -0,0 +1,394 @@
+		{
+			item: {
+				id: "mysticalagriculture:inferium_sword"
+				Count: 1b
+				tag: {
+					Damage: 0
+				}
+			}
+		}
+		{
+			item: {
+				id: "mysticalagriculture:inferium_pickaxe"
+				Count: 1b
+				tag: {
+					Damage: 0
+				}
+			}
+		}
+		{
+			item: {
+				id: "mysticalagriculture:inferium_axe"
+				Count: 1b
+				tag: {
+					Damage: 0
+				}
+			}
+		}
+		{
+			item: {
+				id: "mysticalagriculture:inferium_shovel"
+				Count: 1b
+				tag: {
+					Damage: 0
+				}
+			}
+		}
+		{
+			item: {
+				id: "mysticalagriculture:inferium_scythe"
+				Count: 1b
+				tag: {
+					Damage: 0
+				}
+			}
+		}
+		{ item: "mysticalagriculture:inferium_essence", count: 8, random_bonus: 8, weight: 10 }
+		{ item: "minecraft:cooked_beef", count: 2, random_bonus: 2, weight: 10 }
+		{ item: "minecraft:cooked_porkchop", count: 2, random_bonus: 2, weight: 10 }
+		{ item: "minecraft:cooked_chicken", count: 3, random_bonus: 3, weight: 10 }
+		{ item: "minecraft:oak_log", count: 8, random_bonus: 8, weight: 15 }
+		{ item: "minecraft:stone", count: 4, random_bonus: 8, weight: 10 }
+		{ item: "minecraft:quartz", count: 2, random_bonus: 2, weight: 3 }
+		{ item: "minecraft:feather", count: 4, random_bonus: 4, weight: 7 }
+		{ item: "minecraft:blaze_rod", random_bonus: 1, weight: 3 }
+		{ item: "minecraft:ender_pearl", count: 2, random_bonus: 1, weight: 3 }
+		{ item: "minecraft:bucket", weight: 7 }
+		{ item: "functionalstorage:oak_2", random_bonus: 2, weight: 5 }
+		{ item: "functionalstorage:oak_4", random_bonus: 2, weight: 5 }
+		{ item: "ironfurnaces:iron_furnace", weight: 10 }
+		{ item: "ironfurnaces:gold_furnace", weight: 3 }
+		{
+			item: {
+				id: "alltheores:iron_ore_hammer"
+				Count: 1b
+				tag: {
+					Damage: 0
+				}
+			}
+			weight: 3
+		}
+		{ item: "minecraft:honeycomb", random_bonus: 3, weight: 7 }
+		{ item: "minecraft:honey_bottle", random_bonus: 2, weight: 7 }
+		{ item: "productivebees:honey_treat", random_bonus: 2, weight: 5 }
+		{ item: "minecraft:beehive", weight: 7 }
+		{ item: "productivebees:advanced_oak_beehive", weight: 5 }
+		{
+			item: {
+				id: "buildinggadgets:gadget_building"
+				Count: 1b
+				tag: {
+					state: {
+						serializer: "buildinggadgets:dummy_serializer"
+						state: {
+							Name: "minecraft:air"
+						}
+						data: { }
+					}
+				}
+			}
+		}
+		{ item: "ae2:flawless_budding_quartz" }
+		{
+			command: "/sgear_random_gear @p silentgear:hammer 2"
+			player_command: true
+			type: "command"
+			weight: 5
+		}
+		{
+			command: "/sgear_random_gear @p silentgear:hammer 3"
+			player_command: true
+			type: "command"
+			weight: 3
+		}
+		{
+			command: "/sgear_random_gear @p silentgear:pickaxe 2"
+			player_command: true
+			type: "command"
+			weight: 7
+		}
+		{
+			command: "/sgear_random_gear @p silentgear:pickaxe 3"
+			player_command: true
+			type: "command"
+			weight: 5
+		}
+		{
+			command: "/sgear_random_gear @p silentgear:paxel 3"
```

Comment:
Ideally, these commands should have `@s` instead of `@p`.

Reply:
I'm not sure exactly how this command is going to play out for multiplayer servers. For single player, this will work. However, on a multiplayer server, this command will give it to the closest player when the command is ran. There is an option to toggle it as "Run as player", but will that not require the player to have the permissions to be able to run it? Or does it just have the server run an execute command as the player instead?

Reply:
After some testing, the player will not be able to get the item with the command at all if the option "Run as player" is selected, unless they are an admin/op/enabled cheats.

If the command is changed to @s then the server will run the command, and the player will not get the item. I'm hoping that the @p will listen to the quest command being ran where the player accepts it, and it will give the right player the item.